bodhi/client/bindings.py

```diff
@@ -855,6 +856,66 @@ def get_releases(self, **kwargs):
         """
         return self.send_request('releases/', verb='GET', params=kwargs)
 
+    @errorhandled
+    def save_release(self, **kwargs):
+        """
+        Save a new or edited release.
+
+        Args:
+            kwargs (dict): The parameters to send with the request.
+        """
+        res = self.send_request('releases/', verb='POST', auth=True,
+                                data=kwargs)
+
+        if 'errors' in res:
+            self.print_errors(res)
+
+        else:
+            print("Saved release:")
+            self.print_release(res)
+
+        return res
+
+    def print_release(self, release, verbose=False):
+        """
+        Print a given release to the terminal.
+
+        Args:
+            release (munch.Munch): The release to be printed.
+            verbose (bool): If True, also prints critical path packages
+                            for the release.
+        """
+        print("  Name:                %s" % release['name'])
+        print("  Long Name:           %s" % release['long_name'])
+        print("  Version:             %s" % release['version'])
+        print("  Branch:              %s" % release['branch'])
+        print("  ID Prefix:           %s" % release['id_prefix'])
+        print("  Dist Tag:            %s" % release['dist_tag'])
+        print("  Stable Tag:          %s" % release['stable_tag'])
+        print("  Testing Tag:         %s" % release['testing_tag'])
+        print("  Candidate Tag:       %s" % release['candidate_tag'])
+        print("  Pending Signing Tag: %s" % release['pending_signing_tag'])
+        print("  Pending Testing Tag: %s" % release['pending_testing_tag'])
+        print("  Pending Stable Tag:  %s" % release['pending_stable_tag'])
+        print("  Override Tag:        %s" % release['override_tag'])
+        print("  State:               %s" % release['state'])
+
+        if verbose and 'critpath_pkgs' in release:
+            critpath_pkgs = ",".join([pkg['name'] for pkg in release['critpath_pkgs']])
+            print("  Critpath Packages:   %s" % critpath_pkgs)
+
+    def print_errors(self, data):
+        """
+        Print errors to the terminal and exit with code 1.
+
+        Args:
+            errors (munch.Munch): The errors to be formatted and printed.
+        """
+        for error in data['errors']:
+            print("ERROR: %s" % error['description'])
+
+        sys.exit(1)
```

Comment:
The bindings should never call ```exit()``` because they can be used by long-running services and we don't want to exit on their behalf.

Also, it is surprising for a function called "print" to exit.